themes/theme-ios11/components/Headline/style.js

```diff
@@ -1,12 +1,27 @@
 import { css } from 'glamor';
 import variables from 'Styles/variables';
 
-const headline = css({
-  fontSize: 18,
-  margin: `${variables.gap.big * 2}px 0 ${variables.gap.big}px`,
-  textAlign: 'center',
+const headline = {
+  fontWeight: 700,
+  lineHeight: 1.17,
+  wordBreak: ['keep-all', 'break-word'],
+  hyphens: 'auto',
+  display: 'inline-block',
+};
+
+const large = css({
+  ...headline,
```

Comment:
`css()` can receive multiple parameters. Please use it like this instead of spreading another object it. Leave it to glamor to handle this.